gqlTypes.tsx

```diff
@@ -0,0 +1,175 @@
+export type Maybe<T> = T | null;
+export type InputMaybe<T> = Maybe<T>;
+export type Exact<T extends { [key: string]: unknown }> = {
+  [K in keyof T]: T[K];
+};
+export type MakeOptional<T, K extends keyof T> = Omit<T, K> & {
+  [SubKey in K]?: Maybe<T[SubKey]>;
+};
+export type MakeMaybe<T, K extends keyof T> = Omit<T, K> & {
+  [SubKey in K]: Maybe<T[SubKey]>;
+};
+/** All built-in and custom scalars, mapped to their actual values */
+export type Scalars = {
+  ID: string;
+  String: string;
+  Boolean: boolean;
+  Int: number;
+  Float: number;
+  /** A Date object, transporated as a string */
+  Date: any;
+  /** A DateTime object, transporated as a string */
+  DateTime: any;
```

Comment:
When I regenerate locally, the comments on line 19 and 21 get dropped. Not sure why that is, because I see them in the schema file. I generated with `SCHEMA_PATH=graphql.schema.json yarn graphql:codegen`.

Reply:
See above ^^. The JSON schema here is actually generated FROM the real schema in the warehouse repo. You should use the `schema.graphql` file in the warehouse repo to generate things from, not this JSON representation of the schema.